database/migrations/2023_02_04_184426_extend_oauth_clients_with_custom_data.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('oauth_clients', function (Blueprint $table) {
+            $table->string('privacy_policy_url')->nullable();
+            $table->boolean('webhooks_enabled')->default(false);
+            $table->string('authorized_webhook_url')->nullable();
```

Comment:
```suggestion
            $table->string('authorized_webhook_url')->nullable()->after('redirect');
            $table->string('privacy_policy_url')->nullable()->after('redirect');
            $table->boolean('webhooks_enabled')->default(false)->after('redirect');
```

Just reordering, that the columns are created after `redirect` and in the order from bottom to top. (yes, bottom to top... ._.)